cmd/frontman/main.go

```diff
@@ -315,6 +315,15 @@ func handleFlagInputOutput(inputFile string, outputFile string, oneRunOnlyMode b
 
 func handleFlagOneRunOnlyMode(fm *frontman.Frontman, oneRunOnlyMode bool, inputFile string, output *os.File, interruptChan chan struct{}) {
 	if oneRunOnlyMode {
+		if err := fm.HealthCheck(); err != nil {
+			fm.HealthCheckPassedPreviously = false
+			log.WithError(err).Errorln("Health checks are not passed. Skipping other checks.")
+			return
+		} else if !fm.HealthCheckPassedPreviously {
```

Comment:
So my feedback regarding simplifying the code is just ignored?

Reply:
No, this has been changed according to your feedback:

> The check is against fm.HealthCheckNotPassed NotPassed but the warning then says All health checks are positive This looks odd.

I replaced the variable to avoid double-negotiation, now there is only one.

Reply:
Same discussion as in other places: I talk about the unnecessary code nesting

Reply:
Should be resolved in 2180b23ce68e21f671a11301733ac42517a763a3